final-progII/src/presentation/Menu.java

```diff
@@ -151,7 +149,7 @@ public void actionPerformed(ActionEvent e) {
 		
 		listPendings_menuItem.addActionListener(new ActionListener() {
 			public void actionPerformed(ActionEvent e) {
-				master__cardLayout.show(master__panel, "LIST_STUDENTS_PENDINGS");
+				master__cardLayout.show(master__panel, "name_120454282608500");
```

Comment:
Why did you change the name of the panel?

Reply:
because in the class screen was named like that.